internal/server/caseload.go

```diff
@@ -62,6 +66,28 @@ func caseload(client CaseloadClient, tmpl Template) Handler {
 			return RedirectError(page.CreateUrlBuilder().GetTeamUrl(app.SelectedTeam))
 		}
 
+		if r.Method == http.MethodPost {
+			err := r.ParseForm()
+			if err != nil {
+				return err
+			}
+
+			assignTeam := r.FormValue("assignTeam")
+			newAssigneeId, err := getAssigneeIdForTask(assignTeam, r.FormValue("assignCM"))
+			if err != nil {
+				return err
+			}
+
+			selectedClients := r.Form["selected-clients"]
+
+			assigneeDisplayName, err := client.ReassignClientToCaseManager(ctx, newAssigneeId, selectedClients)
```

Comment:
I was thinking that this would be the cleanest way and quicker then looping through the assignees in workflow in less I have misunderstood what you mean